pkg/state/snapshot_storage.go

```diff
@@ -0,0 +1,40 @@
+package state
+
+import (
+	"github.com/wavesplatform/gowaves/pkg/proto"
+)
+
+type snapshotsAtHeight struct {
+	hs     *historyStorage
+	scheme proto.Scheme
```

Comment:
It's uninitialized. Pass `scheme` as an argument for `snapshotsAtHeight.snapshots` method.